src/core/state.go

```diff
@@ -1136,7 +1150,7 @@ func (state *BuildState) queueTargetAsync(target *BuildTarget, forceBuild, build
 		// Wait for these targets to actually build.
 		if building {
 			for _, t := range target.Dependencies() {
-				t.WaitForBuild()
+				t.WaitForBuild(target.Label)
```

Comment:
What's the reason for this change? (Just cuz it sticks out as not just a debugging change)

Reply:
I need the dependency information so I can add it to the log message